### DISCUSSION TITLE
Inquiry for Citation Information.

### DISCUSSION THREAD

**Discussion by UIowaJinCho** [MahApps/MahApps.Metro#4521] [Q&A]
Hello,

To whom it may concern:

I recently developed some research software with MahApps and I was wondering if you have a specific requirement for citing MahApps.

If you have any information that might help me, please let me know.

Thank you very much for your time.

**Comment by punker76** [comment]
A special citation is not necessary as long as it does not violate the [license](https://github.com/MahApps/MahApps.Metro?tab=MIT-1-ov-file#).
